recipe/reset_from_gitlab_artifact.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Deployer;
+use SourceBroker\DeployerExtendedDatabase\Utility\ConsoleUtility as DatabaseConsoleUtility;
```

Comment:
Why the alias?

Reply:
There was a conflict when configuring the task for a single project in deploy.php - SourceBroker actually has two ConsoleUtilities with different capabilities:
```
use SourceBroker\DeployerExtendedTypo3\Utility\ConsoleUtility;
use SourceBroker\DeployerExtendedDatabase\Utility\ConsoleUtility as DatabaseConsoleUtility;
```

Same issue as here: https://github.com/xima-media/xima-deployer-extended-typo3/blob/60360e6e79eb75f7b72fa2e85680099a27de612c/recipe/check_requirements.php#L5-L6

However, in this task only SourceBroker\DeployerExtendedDatabase\Utility\ConsoleUtility is required so it is possible to not use an alias at all.